index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Sahal</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700;900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="navigation">
+    <a href="#"><img src="images/cabsal.png" id="logo" alt="my logo"></a>
+    <a href="#"><img src="images/Icon - Menu.png" id="menu" alt="Menu"></a> 
+  </nav>
```

Comment:
- [ ] It is required that this header should have a container section called `header` tag. The header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. You can check this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
The expected structure should be like below.

```
 <header class="navigation">
    <a href="#">
      <img src="images/cabsal.png" id="logo" alt="my logo">
    </a>
    <nav>
      <a href="#">
        <img src="images/Icon - Menu.png" id="menu" alt="Menu"> 
      </a>
    </nav> 
  </header>
```